### PR TITLE
Added size attribute to MultiTermsAggregation

### PR DESCRIPTION
**Pull request by fabiankopatschek**
### Description

Added size attribute to MultiTermsAggregation to retreive other hit sizes than the default value of the attribute.
Code for the added attribute was taken from TermsAggregation class.

### Issues Resolved

- https://github.com/opensearch-project/opensearch-java/issues/534 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: java-client/src/main/java/org/opensearch/client/opensearch/_types/aggregations/MultiTermsAggregation.java [hunk 1]
```diff
@@ -37,25 +37,31 @@
 import org.opensearch.client.json.JsonpMapper;
 import org.opensearch.client.json.ObjectBuilderDeserializer;
 import org.opensearch.client.json.ObjectDeserializer;
+import org.opensearch.client.opensearch._types.aggregations.BucketSortAggregation.Builder;
```
Comment (reta) [inline java-client/src/main/java/org/opensearch/client/opensearch/_types/aggregations/MultiTermsAggregation.java:40]: Does not look like the right import

**Comment by reta** [comment]
Thanks for contribution @fabiankopatschek , could you please add a test cases under `AbstractAggregationRequestIT`? Thank you.

**Comment by dblock** [comment]
Also this should go against `main`, and we can backport it after we're good there.

File: java-client/src/main/java/org/opensearch/client/opensearch/_types/aggregations/MultiTermsAggregation.java (hunk 1 above)
Comment (fabiankopatschek) [reply java-client/src/main/java/org/opensearch/client/opensearch/_types/aggregations/MultiTermsAggregation.java]: Thanks for the finding. I removed this line.

**Comment by fabiankopatschek** [comment]
> Thanks for contribution @fabiankopatschek , could you please add a test cases under `AbstractAggregationRequestIT`? Thank you.

I really would like to add a test case. But unfortunately i'm not able to run the integration tests. 
`31 tests completed, 24 failed, 7 skipped`

The AggregationRequestIT for example failes with this message:

> java.lang.RuntimeException: found jar hell in test classpath
	at org.opensearch.bootstrap.BootstrapForTesting.<clinit>(BootstrapForTesting.java:118)
	...

Any tips on how to fix this?

**Comment by reta** [comment]
> Any tips on how to fix this?

Are you running with Eclipse? Or gradle?

**Comment by fabiankopatschek** [comment]
> > Any tips on how to fix this?
> 
> Are you running with Eclipse? Or gradle?

I'm running the tests in the shell with gradle.
I've noticed that the integration test for this MR is having the same trouble.
I encounter the same problems with the main branch as well without any changes from me.

**Comment by reta** [comment]
> I've noticed that the integration test for this MR is having the same trouble.

You are very right, the issue seems to be caused by latest software.amazon.awssdk 2.20.141 (which is imported using version range). Created an issue for it here https://github.com/opensearch-project/opensearch-java/issues/615

**Comment by fabiankopatschek** [comment]
I migrated the code changes to the main branch as [dblock](https://github.com/dblock) suggested.
I also added an Integration Test as [reta](https://github.com/reta) suggested.
There is a new pull request #627 for the changes.

**Pull request closed without merging**